jsonschema-hyperschema.xml

```diff
@@ -1384,37 +1385,196 @@ Link: <https://schema.example.com/entry> rel=describedBy
-                    could also have used "hrefRequired" but with "required" in "hrefSchema"
-                    it is not strictly necessary.  Providing "hrefRequired" without marking
+                    could also have used "templateRequired" but with "required" in "hrefSchema"
+                    it is not strictly necessary.  Providing "templateRequired" without marking
                     "id" as required in "hrefSchema" would lead to errors, as client input
                     is the only possible source for resolving this link.
                 </t>
             </section>
-            <section title="Collections">
-                <t><cref>
-                    Reciprocal collection/item relations
-                    Pagination: fixed links vs jumping to an arbitrary offset
-                    Using "anchorPointer" and "hrefPointer"
-                    Discovering ordered links
-                    Multiple self links (for the collection and each item)
-                </cref></t>
-            </section>
-            <section title="Complex URI Template input">
-                <t><cref>
-                    Pre-populated input data from the instance.
-                    Forbidding input and requiring instance data.
-                    Requiring input and forbidding instance data.
-                </cref></t>
+            <section title="Submitting a payload and accepting URI input">
+                <t>
+                    This example covers using the "submission" fields for non-representation
+                    input, as well as using them alongside of resolving the URI Template with
+                    input.  Unlike HTML forms, which require either constructing a URI or
+                    sending a payload, but do not allow not both at once, JSON Hyper-Schema can
+                    describe both sorts of input to for the same operation on the same link.
+                </t>
+                <t>
+                    The "submissionSchema" and "submissionMediaType" fields are for
+                    describing payloads that are not representations of the target resource.
+                    When used with "http(s)://" URIs, they generally refer to a POST request
+                    payload, as seen in the <xref target="HTTP">appendix on HTTP usage</xref>.
+                </t>
+                <t>
+                    In this case, we use a "mailto:" URI, which, per
+                    <xref target="RFC6068">RFC 6068, Section 3"</xref>, does not provide any
+                    operation for retrieving a resource.  It can only be used to construct
+                    a message for sending.  Since there is no concept of a retrievable,
+                    replaceable, or deletable target resource, "targetSchema" and
+                    "targetMediaType" are not used.  Non-representation payloads are
+                    described by "submissionSchema" and "submission MediaType".
```

Comment:
typo: "submission MediaType".

Also, the last sentence seems superfluous with respect to the next paragraph (and also the first sentence of this section).